model/task/task.go

```diff
@@ -455,13 +455,14 @@ func (tr TestResult) GetLogURL(viewer evergreen.LogViewer) string {
 	case evergreen.LogViewerLobster:
 		// Evergreen-hosted lobster does not support external logs nor
 		// logs stored in the database.
-		if tr.URL != "" || tr.URLRaw != "" || tr.LogId != "" {
-			return ""
-		}
-		for _, url := range deprecatedLobsterURLs {
-			if strings.Contains(tr.URL, url) {
-				return strings.Replace(tr.URL, url, root+"/lobster", 1)
+		if tr.URL != "" || tr.URLRaw != "" {
```

Comment:
was it purposeful to drop LogId from this? (just checking i have zero context here)

Reply:
It's not being used so I think it's fine.